CompileDefinitions.cmake

```diff
@@ -0,0 +1,177 @@
+if (WIN32)
```

Comment:
Please use only lowercase letters for filenames. That is a convention that we have picked for coreclr at the beginning.